index.css

```diff
@@ -0,0 +1,84 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
```

Comment:
This `index.css` stylesheet is not a CSS module (and I don't think it could be one), so these classes are all global. Also, this stylesheet is outside the regular hierarchy of the code (e.g. in the `src` folder), meaning it will probably be forgotten by future devs. So, we should aim to only include in this file styles that are specific to the non-React code and that will not interact with the React stlying at all.

That means that:
1. Each class needs to be very specifically named (e.g. `splash-screen-icon`)
2. We should not use the `*` selector, because it selects every element on the entire page, which it should not do.